packages/lesswrong/lib/collections/users/custom_fields.js

```diff
@@ -147,6 +147,29 @@ Users.addField([
     }
   },
 
+  /**
+    bannedUserIds: users who are not allowed to comment on this user's posts
+  */
+
+  {
+    fieldName: 'bannedUserIds',
```

Comment:
Seems to me like it should very likely resolve to an array of users, in the way all of our userId fields do

Reply:
Can we do that as part of a separate pass that also updates the UI of the bannedUserIds list on posts and users?